src/test/java/posters/pageobjects/pages/user/OrderHistoryPage.java

```diff
@@ -18,9 +18,10 @@
 
     @Override
     @Step("ensure this is an order history page")
-    public void isExpectedPage()
+    public OrderHistoryPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.